src/pages/index.js

```diff
@@ -181,7 +178,7 @@ Promise.all([api.getUserInformation(), api.getInitialCards()])
     userInfo.setUserInfo(userData);
     userInfo.setUserProfileImage(userData);
     initialCards.forEach((card) => addCard(card));
+    // Render cards
+    cardSection.renderItems();
```

Comment:
You haven't corrected my remark about `renderItems ` https://github.com/NoobCoder815/se_project_aroundtheus/pull/1#discussion_r1373704238 

It should be like this



```js
    // Render cards
    cardSection.renderItems(initialCards);
```


`renderItems` should accept the array from now on to be able to render it